docs/installation.rst

```diff
@@ -198,10 +174,10 @@ To start the dev server, run ``./manage.py runserver``, then open up
 ``http://localhost:8000``. If everything's working, you should see
 the MDN home page!
 
-You might need to first set ``LC_CTYPE`` if you're on Mac OS X until
-`bug 754728 <https://bugzilla.mozilla.org/show_bug.cgi?id=754728>`_ is fixed::
+You might need to first set ``LC_CTYPE`` in your ``.env`` file if you're on
+Mac OS X until `bug 754728 <https://bugzil.la/754728>`_ is fixed::
```

Comment:
Makes sense. Looks like the bug is "fixed" by including the docs. :neutral_face: